x-pack/solutions/observability/plugins/slo/ui_tests/fixtures/page_objects/annotations_app.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { expect, ScoutPage } from '@kbn/scout-oblt';
+
+export class AnnotationsApp {
+  constructor(private readonly page: ScoutPage) {}
+
+  async goto() {
+    await this.page.gotoApp('observability-overview', {});
+    await expect(this.page.getByText('Annotations')).toBeVisible();
+    await this.page.click('text=Annotations');
```

Comment:
Can we add a similar check to make sure annotations screen (at least without content) is loaded?